src/components/Contact/ContactForm.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import { film, header } from "../../utils/mockData";
+import { patchData } from '../../utils/apiCalls';
+
+import TextField from '@mui/material/TextField';
+import FormControl from '@mui/material/FormControl';
+import Button from '@mui/material/Button';
+
+const ContactForm = () => {
+  const [name, setName] = useState<string>('')
+  const [phoneNum, setPhoneNum] = useState<string>('')
+  const [email, setEmail] = useState<string>('')
+  const [company, setCompany] = useState<string>('')
+
+  const handleSubmit = () => {
+    let currentContact = {
+      contact_name: name, 
+      contact_number: phoneNum, 
+      contact_email: email, 
+      company_name: company
+    }
+    patchData(currentContact, 14)
+    console.log('currentContact: ', currentContact)
+  }
+
+  return (
+    <section className='contact-form-container'>
+      <h2>Contact</h2>
+      <FormControl sx={{ m: 1, minWidth: 120 }}>
```

Comment:
Ok, so this is the styled components? Very cool

Reply:
Yes, Rachel. We can walk through this tomorrow.